sqldelight-studio-plugin/src/main/kotlin/com/squareup/sqldelight/intellij/lang/SqlDelightCompletionContributor.kt

```diff
@@ -99,15 +99,13 @@ private class SqlDelightCompletionProvider : CompletionProvider<CompletionParame
             }
         is ResolutionError.ColumnNameNotFound -> error.availableColumns
             .flatMap { it.columnNames() }
-            .filterNotNull()
```

Comment:
same with these

Reply:
`error.availableColumns` returns a `Collection<Result>`, `result.columnNames` returns `List<String>` in which there cannot be null.